### PR TITLE
Add logging to Go getting-started example

### PR DESCRIPTION
**Pull request by MrAlias**
Resolve https://github.com/open-telemetry/opentelemetry-go/issues/5058

cc @open-telemetry/go-approvers

### CODE REVIEW

File: content/en/docs/languages/go/getting-started.md [hunk 1]
```diff
@@ -17,7 +17,7 @@ way that [traces][] and [metrics][] are emitted to the console.
 
 Ensure that you have the following installed locally:
 
-- [Go](https://go.dev/)
+- [Go](https://go.dev/) `v1.22` or greater
```
Comment (pellared) [inline content/en/docs/languages/go/getting-started.md:20]: nit:

```suggestion
- [Go](https://go.dev/) `1.22` or greater
```

Go does not name their versions with `v` prefix.

**Review by pellared — APPROVED** [review]
Just one nit comment.

**Review by pellared — COMMENTED** [review]
Shouldn't we mark that logger part is experimental?

**Comment by svrnm** [comment]
Update line 13-14:

```
You will learn how you can instrument a simple application manually, in such a
way that [traces][] and [metrics][] are emitted to the console.
```

add "logs"

File: content/en/docs/languages/go/getting-started.md [hunk 2]
```diff
@@ -17,7 +17,7 @@ way that [traces][] and [metrics][] are emitted to the console.
 
 Ensure that you have the following installed locally:
 
-- [Go](https://go.dev/)
+- [Go](https://go.dev/) `1.22` or greater
```
Comment (chalin) [inline content/en/docs/languages/go/getting-started.md:20]:
```suggestion
- [Go](https://go.dev/) 1.22 or greater
```

File: content/en/docs/languages/go/getting-started.md [hunk 3]
```diff
@@ -646,11 +688,86 @@ it is a child of the previously mentioned span.
 
 </details>
 
-Refresh the <http://localhost:8080/rolldice> page a few times, and then either
-wait for a little bit or terminate the app and you'll see metrics as in the
-console output. You'll see the `dice.rolls` metric emitted to the console, with
-separate counts for each roll value, as well as the HTTP metrics generated by
-the instrumentation library.
+Along with the trace, a log messages will be emitted to the console.
```
Comment (chalin) [inline content/en/docs/languages/go/getting-started.md:691]:
```suggestion
Along with the trace, log messages are emitted to the console.
```

File: content/en/docs/languages/go/getting-started.md [hunk 4]
```diff
@@ -646,11 +688,86 @@ it is a child of the previously mentioned span.
...
+    "SchemaURL": ""
+  },
+  "DroppedAttributes": 0
+}
+```
+
+</details>
+
+Refresh the <http://localhost:8080/rolldice/Alice> page a few times, and then
+either wait for a little bit or terminate the app and you'll see metrics as in
```
Comment (chalin) [inline content/en/docs/languages/go/getting-started.md:767]:
```suggestion
either wait a little or terminate the app and you'll see metrics as in
```

**Review by chalin — APPROVED** [review]
Copyedits. Otherwise, LGTM. I'll let other check the code.

**Pull request merged**